Doc/library/sqlite3.rst

```diff
@@ -1223,6 +1251,45 @@ Connection objects
 
       .. versionadded:: 3.11
 
+   .. attribute:: autocommit
+
+      This attribute controls :pep:`249`-compliant transaction behaviour.
+      :attr:`!autocommit` has three allowed values:
+
+      * ``False``: Select :pep:`249`-compliant transaction behaviour,
+        implying that :mod:`!sqlite3` ensures a transaction is always open.
+        Use :meth:`commit` and :meth:`rollback` to close transactions.
+
+        This is the recommended value of :attr:`!autocommit`.
+
+      * ``True``: Use SQLite's `autocommit mode`_.
+        :meth:`commit` and :meth:`rollback` have no effect in this mode.
+
+      * :data:`LEGACY_TRANSACTION_CONTROL`:
+        Pre-Python 3.12 (non-:pep:`249`-compliant) transaction control.
+        See :attr:`isolation_level` for more details.
+
+        This is currently the default value of :attr:`!autocommit`.
+
+      Changing :attr:`!autocommit` to ``False`` will open a new transaction,
+      and changing it to ``True`` will commit any pending transaction.
+
+      See :ref:`sqlite3-transaction-control-autocommit` for more details.
+
+      .. note::
+
+         The :pep:`249`-compliant autocommit feature and the
+         SQLite `autocommit mode`_ are two related but distinct concepts.
+         Use :attr:`in_transaction` to query the low-level SQLite
+         autocommit mode.
```

Comment:
FYI, IMO this note is a better version of the one in the explanation, that leaves the What Ifs out of it and just clarifies the main points of act. I'd also suggest either replacing that one with this one, or just leave this one here and removing that one.

Also, it looks rather odd to have the two notes back to back, so I'd suggest moving it to under the `True` bullet where SQLIte's autocommit mode is mentioned.